.github/workflows/centos_8.yml

```diff
@@ -18,6 +18,9 @@ jobs:
       fail-fast: false
 
     steps:
+      # must be before the main job to get needed environment,
+      # used to get CI_ACTION_REF_NAME value
+      - uses: FranzDiebold/github-env-vars-action@v2.1.0
```

Comment:
Does not it solve the same task as [$GITHUB_ENV](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable)?